boards/stm32l476rg/src/tasks/pneumatics.rs

```diff
@@ -0,0 +1,11 @@
+use embassy_time::{Duration, Timer};
+
+#[embassy_executor::task]
+pub async fn control_pneumatics() -> ! {
+    let p = embassy_stm32::init(Default::default());
+    let mut pneumatics_control = PneumaticsControl::new();

```

Comment:
Just need to complete this by creating two GPIO inputs and trying to create the pneumatics control struct